janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -648,16 +648,12 @@ public boolean apply(@Nullable String s) {
             "Number of times the system attempts to acquire a lock before giving up and throwing an exception",
             ConfigOption.Type.MASKABLE, 3);
 
-    /**
-     * The number of milliseconds the system waits for a lock application to be acknowledged by the storage backend.
-     * Also, the time waited at the end of all lock applications before verifying that the applications were successful.
-     * This value should be a small multiple of the average consistent write time.
-     */
```

Comment:
Why this comment was removed?

Reply:
I removed it because it simply duplicates the code without any extra information.

Reply:
Oh, you are right. I somehow checked added lines only without checking unchanged lines. Now I see that it's just a duplication